index.js

```diff
@@ -1,10 +1,30 @@
 'use strict';
-module.exports = function (flag, argv) {
+module.exports = function (flags, argv) {
 	argv = argv || process.argv;
 
+	var contains = false;
+
+	if (Array.isArray(flags)) {
+		var matches = flags.filter(function (flag) {
+			return containsTheTerm(flag, argv);
+		});
+
+		contains = flags.length === matches.length;
+	} else {
+		contains = containsTheTerm(flags, argv);
+	}
+
+	return contains;
```

Comment:
Instead of hoisting `contains`, do this:

```js
if (Array.isArray(flags)) {
    ...
   return flags.length === matches.length;
}

return containsTheTerm(flags, argv);
```